zebra-chain/src/block.rs

```diff
@@ -143,6 +145,22 @@ impl Block {
             .map(|transaction| transaction.orchard_nullifiers())
             .flatten()
     }
+
+    /// Get all the value balances from this block by summing all the value balances
+    /// in each transaction the block has.
+    ///
+    /// `utxos` must contain the utxos of every input in the transaction,
```

Comment:
There's a confusing typo here, but after that's fixed we can merge:
```suggestion
    /// `utxos` must contain the utxos of every input in the block,
```